<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>950</width>
    <height>800</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>CS123 - Final</string>
  </property>
  <widget class="QWidget" name="centralWidget"/>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>950</width>
     <height>19</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionQuit"/>
   </widget>
   <addaction name="menuFile"/>
  </widget>
  <widget class="QDockWidget" name="dockWidget">
   <property name="minimumSize">
    <size>
     <width>100</width>
     <height>160</height>
    </size>
   </property>
   <attribute name="dockWidgetArea">
    <number>1</number>
   </attribute>
   <widget class="QWidget" name="dockWidgetContents">
    <widget class="QGroupBox" name="fboGroupBox">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>0</y>
       <width>221</width>
       <height>91</height>
      </rect>
     </property>
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>0</height>
      </size>
     </property>
     <property name="title">
      <string>Scenes</string>
     </property>
     <widget class="QRadioButton" name="modeScene1">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>30</y>
        <width>81</width>
        <height>22</height>
       </rect>
      </property>
      <property name="text">
       <string>Scene 1</string>
      </property>
      <property name="checked">
       <bool>false</bool>
      </property>
     </widget>
     <widget class="QRadioButton" name="modeScene2">
      <property name="geometry">
       <rect>
        <x>100</x>
        <y>30</y>
        <width>81</width>
        <height>22</height>
       </rect>
      </property>
      <property name="text">
       <string>Scene 2</string>
      </property>
     </widget>
     <widget class="QRadioButton" name="modeScene3">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>60</y>
        <width>81</width>
        <height>22</height>
       </rect>
      </property>
      <property name="text">
       <string>Scene 3</string>
      </property>
      <property name="checked">
       <bool>true</bool>
      </property>
     </widget>
    </widget>
    <widget class="QGroupBox" name="lightIntensityGroup">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>580</y>
       <width>221</width>
       <height>121</height>
      </rect>
     </property>
     <property name="title">
      <string>Light Intensities</string>
     </property>
     <widget class="QSlider" name="l1Slider">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>30</y>
        <width>121</width>
        <height>16</height>
       </rect>
      </property>
      <property name="maximum">
       <number>100</number>
      </property>
      <property name="sliderPosition">
       <number>100</number>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
     <widget class="QLineEdit" name="l1Text">
      <property name="geometry">
       <rect>
        <x>160</x>
        <y>30</y>
        <width>41</width>
        <height>21</height>
       </rect>
      </property>
     </widget>
     <widget class="QSlider" name="l2Slider">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>60</y>
        <width>121</width>
        <height>16</height>
       </rect>
      </property>
      <property name="maximum">
       <number>100</number>
      </property>
      <property name="pageStep">
       <number>10</number>
      </property>
      <property name="sliderPosition">
       <number>100</number>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
     <widget class="QLineEdit" name="l2Text">
      <property name="geometry">
       <rect>
        <x>160</x>
        <y>60</y>
        <width>41</width>
        <height>21</height>
       </rect>
      </property>
     </widget>
     <widget class="QSlider" name="l3Slider">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>90</y>
        <width>121</width>
        <height>16</height>
       </rect>
      </property>
      <property name="maximum">
       <number>100</number>
      </property>
      <property name="value">
       <number>100</number>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
     <widget class="QLineEdit" name="l3Text">
      <property name="geometry">
       <rect>
        <x>160</x>
        <y>90</y>
        <width>41</width>
        <height>21</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="lightLabel1">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>30</y>
        <width>21</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>#1</string>
      </property>
     </widget>
     <widget class="QLabel" name="lightLabel2">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>60</y>
        <width>21</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>#2</string>
      </property>
     </widget>
     <widget class="QLabel" name="lightLabel3">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>90</y>
        <width>21</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>#3</string>
      </property>
     </widget>
    </widget>
    <widget class="QGroupBox" name="featuresGroup">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>100</y>
       <width>221</width>
       <height>241</height>
      </rect>
     </property>
     <property name="title">
      <string>Features</string>
     </property>
     <widget class="QCheckBox" name="cbAO">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>90</y>
        <width>141</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Ambient Occlusion</string>
      </property>
     </widget>
     <widget class="QCheckBox" name="cbDOF">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>120</y>
        <width>131</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Depth of Field</string>
      </property>
     </widget>
     <widget class="QCheckBox" name="cbNM">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>60</y>
        <width>131</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Normal Mapping</string>
      </property>
     </widget>
     <widget class="QSlider" name="apertureSlider">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>170</y>
        <width>131</width>
        <height>16</height>
       </rect>
      </property>
      <property name="minimum">
       <number>1</number>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
     <widget class="QSlider" name="focalSlider">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>210</y>
        <width>131</width>
        <height>16</height>
       </rect>
      </property>
      <property name="minimum">
       <number>4</number>
      </property>
      <property name="maximum">
       <number>80</number>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
     <widget class="QLineEdit" name="apertureText">
      <property name="geometry">
       <rect>
        <x>170</x>
        <y>150</y>
        <width>41</width>
        <height>21</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="focalText">
      <property name="geometry">
       <rect>
        <x>170</x>
        <y>200</y>
        <width>41</width>
        <height>21</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="apertureLabel">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>150</y>
        <width>61</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Aperture</string>
      </property>
     </widget>
     <widget class="QLabel" name="focalLabel">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>190</y>
        <width>131</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Focal Point Distance</string>
      </property>
     </widget>
     <widget class="QCheckBox" name="cbStochastic">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>30</y>
        <width>161</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Stochastic Sampling</string>
      </property>
     </widget>
    </widget>
    <widget class="QGroupBox" name="lightingContribGroup">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>420</y>
       <width>221</width>
       <height>151</height>
      </rect>
     </property>
     <property name="title">
      <string>Lighting Components</string>
     </property>
     <widget class="QCheckBox" name="cbAmbient">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>30</y>
        <width>141</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Ambient</string>
      </property>
     </widget>
     <widget class="QCheckBox" name="cbDiffuse">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>60</y>
        <width>131</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Diffuse</string>
      </property>
     </widget>
     <widget class="QCheckBox" name="cbSpecular">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>90</y>
        <width>131</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Specular</string>
      </property>
     </widget>
     <widget class="QCheckBox" name="cbShadows">
      <property name="geometry">
       <rect>
        <x>110</x>
        <y>30</y>
        <width>131</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Shadows</string>
      </property>
     </widget>
     <widget class="QCheckBox" name="cbReflection">
      <property name="geometry">
       <rect>
        <x>110</x>
        <y>60</y>
        <width>131</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Reflection</string>
      </property>
     </widget>
     <widget class="QCheckBox" name="cbTextures">
      <property name="geometry">
       <rect>
        <x>110</x>
        <y>90</y>
        <width>131</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Textures</string>
      </property>
     </widget>
     <widget class="QCheckBox" name="cbEnviro">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>120</y>
        <width>181</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Environment Cube</string>
      </property>
     </widget>
    </widget>
    <widget class="QGroupBox" name="cameraGroup">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>710</y>
       <width>221</width>
       <height>61</height>
      </rect>
     </property>
     <property name="title">
      <string>Animation</string>
     </property>
     <widget class="QSlider" name="l3Slider_2">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>90</y>
        <width>121</width>
        <height>16</height>
       </rect>
      </property>
      <property name="maximum">
       <number>100</number>
      </property>
      <property name="value">
       <number>100</number>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
     <widget class="QLineEdit" name="l3Text_2">
      <property name="geometry">
       <rect>
        <x>160</x>
        <y>90</y>
        <width>41</width>
        <height>21</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="lightLabel3_2">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>90</y>
        <width>21</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>#3</string>
      </property>
     </widget>
     <widget class="QCheckBox" name="cbAnimation">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>30</y>
        <width>181</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Animation</string>
      </property>
     </widget>
    </widget>
    <widget class="QGroupBox" name="samplesGroup">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>350</y>
       <width>221</width>
       <height>61</height>
      </rect>
     </property>
     <property name="title">
      <string>Number of samples</string>
     </property>
     <widget class="QSlider" name="samplesSlider">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>30</y>
        <width>131</width>
        <height>16</height>
       </rect>
      </property>
      <property name="maximum">
       <number>40</number>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
     <widget class="QLineEdit" name="samplesText">
      <property name="geometry">
       <rect>
        <x>160</x>
        <y>30</y>
        <width>31</width>
        <height>21</height>
       </rect>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
  <action name="actionQuit">
   <property name="text">
    <string>Quit</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Q</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections>
  <connection>
   <sender>actionQuit</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>299</x>
     <y>199</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
